library/build.gradle.kts

```diff
@@ -1,10 +1,14 @@
+import com.vanniktech.maven.publish.SonatypeHost
 import org.jetbrains.kotlin.gradle.ExperimentalKotlinGradlePluginApi
 import org.jetbrains.kotlin.gradle.dsl.JvmTarget
 
+group = "io.github.kotlin"
+version = "1.0.0"
+
 plugins {
     alias(libs.plugins.kotlinMultiplatform)
     alias(libs.plugins.androidLibrary)
-    id("module.publication")
+    alias(libs.plugins.publish)
 }
 
```

Comment:
Are there strong reasons for having configuration before the plugins block?

It's typical to have no code before the plugins block. I think that it's recommended by Gradle. 

```kotlin
plugins {
    alias(libs.plugins.kotlinMultiplatform)
    alias(libs.plugins.androidLibrary)
    alias(libs.plugins.publish)
}

group = "io.github.kotlin"
version = "1.0.0"
```